src/components/Upload/UploadManager.tsx

```diff
@@ -0,0 +1,235 @@
+import {
+  Dialog,
+  DialogContent,
+  IconButton,
+  makeStyles,
+  DialogTitle as MuiDialogTitle,
+  Paper,
+  PaperProps,
+  Typography,
+} from '@material-ui/core';
+import {
+  Close as CloseIcon,
+  Maximize as MaximizeIcon,
+  Minimize as MinimizeIcon,
+} from '@material-ui/icons';
+import clsx from 'clsx';
+import React, {
+  FC,
+  memo,
+  useCallback,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from 'react';
+import Draggable, { DraggableData, DraggableEvent } from 'react-draggable';
+import { useWindowSize } from 'react-use';
+import { useSession } from '../Session';
+import { useUploadManager } from './UploadContext';
+
+const PAPER_WIDTH = 480;
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  root: {
+    pointerEvents: 'none',
+  },
+  paper: {
+    margin: 0,
+    pointerEvents: 'auto',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    width: `${PAPER_WIDTH}px`,
+  },
+  titleContainer: {
+    cursor: 'move',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: spacing(1),
+    '&.collapsed': {
```

Comment:
You can add an empty collapsed class here and then reference it like `&$collapsed`. [MUI](https://github.com/mui-org/material-ui/blob/50c0bbc3a2040c9945f5009a6bd56292d216f236/packages/material-ui/src/InputLabel/InputLabel.js) does this too.

Reply:
I tried this and the styles didn't take, but I'll have to look at it again... maybe I got the syntax wrong

Reply:
You also need to apply it with `classes.collapsed` instead of the `"collapsed"` string constant.

Reply:
I remember now... TypeScript doesn't like this; it can't tell that `classes.collapsed` exists